packages/specification/schemas/components/CommunicationRequest/CommunicationRequest.yaml

```diff
@@ -1,7 +1,8 @@
 type: object
 description: |
   Additional instructions intended for the patient.
-  For repeat prescriptions, a reference must also be provided to the list of repeat medications that the patient is on. 
+  CommunicationRequests should be prefixed with the medication line item the instructions are for.
```

Comment:
I think to be as clear as possible, the text should specify that the ID should be used
```
CommunicationRequests should be prefixed with the medication line item ID that the instructions are for.
```
thoughts?